trino/trino.go

```diff
@@ -100,6 +100,9 @@ var (
 
 	// ErrInvalidResponseType indicates that the server returned an invalid type definition.
 	ErrInvalidResponseType = errors.New("trino: server response contains an invalid type")
+
+	// ErrInvalidProgressCallbackHeader indicates that server did not get valid headers for progress callback
+	ErrInvalidProgressCallbackHeader = errors.New("trino: " + trinoProgressCallbackParam + trinoProgressCallbackPeriodParam + " both must be set when using progress callback")
```

Comment:
nit: `"trino: both " + trinoProgressCallbackParam + " and " + trinoProgressCallbackPeriodParam + " must be set when using progress callback"`